### DISCUSSION TITLE
How to hide a widget when there are no data to display?

### DISCUSSION THREAD

**Discussion by 3pleo** [filamentphp/filament#14259] [Help]
### Package

Panel builder

### Package Version

V3

### How can we help you?

Please how can I conditionally hide a widget when it has no data to display.

I can’t seem to figure out how to use the static canview function to access the getoption function to conditionally determine if there are data and based on this decide on if to display or not.

**Comment by leandrocfe** [comment]
What kind of widget are you using?

**Reply by 3pleo** [reply]
Thanks Leandro,
I’m using Chart Widget & Table widget
